cli/src/utils.rs

```diff
@@ -188,6 +188,17 @@ macro_rules! p_println {
 }
 pub(crate) use p_println;
 
+/// Loads a dotenv file, ignoring potential failure.
+///
+/// We could use `tracing::warn!` here, but that would imply that the dotenv file can't configure
+/// the logging behavior of Foundry.
```

Comment:
... because we would need to register the subscriber before loading `.env`

Reply:
this will now auto load regardless which is a big change.

I think we should consider making this opt-in via Config?

Reply:
not sure... if config is loaded, then dotenv won't take effect on config values? also, not sure why you would have a .env in your project w/o it being related. Not sure

Maybe it's possible to write a Figment provider? Although that would mean things outside the scope of config won't be interpreted (`NO_COLOR` etc)